kinolights.json

```
@@ -27,5 +27,10 @@
         "estimated": ""
       }
     }
+  },
+  "popcorn": {
+    "youtube": ["https://www.youtube.com/@kinolights"],
+    "instagram": ["https://www.instagram.com/kinolights"],
+    "others_video": ["https://youtu.be/Zc3W3xNHXbw?si=c5fOU3FXy92n_iwI"]
```

P2: si는 트래킹정보라서 빼주세요. 주소변환이 한번 일어나니 원본주소를 써줘도 좋겠네요
```suggestion
    "others_video": ["https://www.youtube.com/watch?v=Zc3W3xNHXbw"]
```

---

이건 처음 알았네요 😮 수정했습니다!